### DISCUSSION TITLE
Beginner here, can anyone help me figure out how to use ShairPort-Sync to output audio to bluetooth using Mac?

### DISCUSSION THREAD

**Discussion by SSMalcontent** [mikebrady/shairport-sync#1617] [Q&A]
I have been running circles for a while now, and I can't find much in the way of resources for beginners to the issues I'm trying to address.

I'm running SPS in my Macbook, and I want to output the audio that SPS receives from AirPlay into a Bluetooth Connected Speaker. Problem is, every resource I find detailing how to use Bluetooth with SPS mentions ALSA, which isn't available on MacOS. Then I tried using PulseAudio, which is available using brew, but not with Bluetooth. After some digging I discovered that PipeWire does what PulseAudio-Bluetooth used to do, but I couldn't figure out how to install PipeWire on my machine because it keeps saying it couldn't find dbus-1 on the dependencies, even though I downloaded dbus from brew.

I'm not an expert on any of this, and I'm stumbling in the dark trying to find a solution. The pieces are all here, I just can't seem to figure out how to connect them.

I unfortunately cannot introduce Raspberry Pi into my hardware mix because it's too expensive in my country, so I'm running SPS in my Macbook.

Is there any other way to configure SPS to output straight into bluetooth?

**Comment by mikebrady** [comment]
Thanks for the question. Shairport Sync doesn't output straight to Bluetooth, I'm afraid.

**Reply by Siegum** [reply]
Apologies if this is a little off-topic, because it has nothing to do with Shairport Sync.
I am assuming that your macbook cannot be updated to Monterey ?, because from Monterey and on you can Airplay natively to your Mac (without SPS). In this case, you just choose your bluetooth speakers as audio output of the macbook and voila.

Even if you can not update to Monterey because your macbook is too old, it is often possible to use opencore legacy patcher to install Monterey on older Macs. I have personally updated a Mac Mini from 2010 and a Macbook Air from 2013 to Monterey (and even Ventura) with this tool. I just tried now to select a bluetooth speaker as audio output on the Macbook Air and airplay from my phone to the macbook.....and music flowed from the bluetooth speaker :-)

**Reply by mikebrady** [reply]
This is a very nice solution!
